src/quo2/components/switchers/base_card/view.cljs

```diff
@@ -0,0 +1,28 @@
+(ns quo2.components.switchers.base-card.view
+  (:require [quo.react-native :as rn]
+            [quo2.components.buttons.button.view :as button]
+            [quo2.components.switchers.base-card.style :as style]
+            [quo2.foundations.colors :as colors]))
+
+(defn base-card
+  [{:keys [banner on-press on-close customization-color]} & children]
+  (let [card-ref (atom nil)
+        color-50 (colors/custom-color customization-color 50)]
```

Comment:
let's just pass `customization-color` along, i.e (style/base-container customization-color)` and handle that in there. I don't think we need this variable